### ISSUE TITLE
Not working in React 16

### ISSUE THREAD

**Issue by codler** [nordnet/nordnet-ui-kit#304]
It fails in @\iamstarkov/jss-theme-reactor package where it uses the old style of including PropTypes.
```
ThemeProvider.js:130 Uncaught TypeError: Cannot read property 'node' of undefined
    at createThemeProvider (ThemeProvider.js:130)
    at Object.defineProperty.value (ThemeProvider.js:143)
    at __webpack_require__ (bootstrap 6cd1348b135d35dcd409:19)
    at Object.<anonymous> (ThemeProvider.js:9)
    at Object.defineProperty.value (index.58b54d2abfbab2d4b54a.js:98707)
    at __webpack_require__ (bootstrap 6cd1348b135d35dcd409:19)
    at Object.defineProperty.value (index.js:7)
    at __webpack_require__ (bootstrap 6cd1348b135d35dcd409:19)
    at Object.canDefineProperty (index.js:116)
    at __webpack_require__ (bootstrap 6cd1348b135d35dcd409:19)
```

**Comment by bstream** [comment]
I believe this is fixed by https://github.com/nordnet/nordnet-ui-kit/pull/303?

**Comment by iamstarkov** [comment]
#303 has been released as v1.18.1
